### PR TITLE
Updating docs for OpenVino (version 2023.0.0)

### LINKED ISSUE #136: [DOCS] The OpenVINO is updated to 2023 edition. Related docs should be updated
### Description

See https://github.com/WasmEdge/WasmEdge/pull/2636

The OpenVINO is updated. So we will need to update the corresponding documents.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Mash707**
resolves #136.

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by Mash707** [comment]
@hydai have a look at the changes and let me know what needs to updated.

File: docs/contribute/source/plugin/wasi_nn.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 2
 
 ## Prerequisites
 
-Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#)(2021) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
+Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP)(2023) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
```
Comment (hydai) [inline docs/contribute/source/plugin/wasi_nn.md:9]: The link is wrong. It tried ty use `OS_SYSTEM=WINDOWS`.

File: docs/start/install.md [hunk 2]
```diff
@@ -181,16 +181,14 @@ Then, go to the [WASI-NN PyTorch backend in Rust chapter](../develop/rust/wasinn
 
 `WASI-NN` plug-in with `OpenVINO™` backend allows WasmEdge applications to perform `OpenVINO™` model inference. To install WasmEdge with `WASI-NN OpenVINO™ backend` plug-in on Linux, please use the `--plugins wasi_nn-openvino` parameter when [running the installer command](#generic-linux-and-macos).
 
-The `WASI-NN` plug-in with `OpenVINO™` backend depends on the `OpenVINO™` C library to perform AI/ML computations. [OpenVINO™](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#)(2021) dependencies. The following instructions are for Ubuntu 20.04 and above.
+The `WASI-NN` plug-in with `OpenVINO™` backend depends on the `OpenVINO™` C library to perform AI/ML computations. [OpenVINO™](https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP)(2023) dependencies. The following instructions are for Ubuntu 20.04 and above.
```
Comment (hydai) [inline docs/start/install.md:184]: Wrong link

File: i18n/zh/docusaurus-plugin-content-docs/current/contribute/source/plugin/wasi_nn.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 2
 
 ## Prerequisites
 
-Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#)(2021) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
+Currently, WasmEdge used OpenVINO™ or PyTorch as the WASI-NN backend implementation. For using WASI-NN on WasmEdge, you need to install [OpenVINO™](https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP)(2023) or [PyTorch 1.8.2 LTS](https://pytorch.org/get-started/locally/) for the backend.
```
Comment (hydai) [inline i18n/zh/docusaurus-plugin-content-docs/current/contribute/source/plugin/wasi_nn.md:9]: Ditto.

File: i18n/zh/docusaurus-plugin-content-docs/current/start/install.md [hunk 4]
```diff
@@ -181,16 +181,14 @@ Then, go to the [WASI-NN PyTorch backend in Rust chapter](../develop/rust/wasinn
 
 `WASI-NN` plug-in with `OpenVINO™` backend allows WasmEdge applications to perform `OpenVINO™` model inference. To install WasmEdge with `WASI-NN OpenVINO™ backend` plug-in on Linux, please use the `--plugins wasi_nn-openvino` parameter when [running the installer command](#generic-linux-and-macos).
 
-The `WASI-NN` plug-in with `OpenVINO™` backend depends on the `OpenVINO™` C library to perform AI/ML computations. [OpenVINO™](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux.html#)(2021) dependencies. The following instructions are for Ubuntu 20.04 and above.
+The `WASI-NN` plug-in with `OpenVINO™` backend depends on the `OpenVINO™` C library to perform AI/ML computations. [OpenVINO™](https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP)(2023) dependencies. The following instructions are for Ubuntu 20.04 and above.
```
Comment (hydai) [inline i18n/zh/docusaurus-plugin-content-docs/current/start/install.md:184]: Ditto

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/contribute/source/plugin/wasi_nn.md (hunk 1 above)
Comment (Mash707) [reply docs/contribute/source/plugin/wasi_nn.md:9]: https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=DEV_TOOLS&OP_SYSTEM=WINDOWS&VERSION=v_2023_0_1&DISTRIBUTION=PIP 
The link is working.

Comment (hydai) [reply docs/contribute/source/plugin/wasi_nn.md:9]: No, it doesn't.

Your link shows:
CleanShot 2023-07-20 at 00 37 16

However, according to your document, it should be `https://docs.openvino.ai/2023.0/openvino_docs_install_guides_overview.html?ENVIRONMENT=RUNTIME&OP_SYSTEM=LINUX&VERSION=v_2023_0_1&DISTRIBUTION=APT`

CleanShot 2023-07-20 at 00 38 23

**Review by hydai — APPROVED** [review]

**Pull request merged**